### PR TITLE
Emit event on status-change

### PR DESCRIPTION
**Pull request by georgehrke**

### CODE REVIEW

File: apps/user_status/src/store/userStatus.js [hunk 1]
```diff
@@ -145,64 +146,92 @@ const actions = {
...
+	 * @param {Object} vuex.state The Vuex state object
 	 * @param {Object} data The data destructuring object
 	 * @param {String} data.statusType The new status type
 	 * @returns {Promise<void>}
 	 */
-	async setStatus({ commit }, { statusType }) {
+	async setStatus({ commit, state }, { statusType }) {
 		await setStatus(statusType)
 		commit('setStatus', { statusType })
+		emit('user_status:status.updated', {
```
Comment (nickvergessen) [inline apps/user_status/src/store/userStatus.js:157]: if we add the user, we could use the same event for other user status-updates as well?

Comment (georgehrke) [reply apps/user_status/src/store/userStatus.js:157]: Sounds reasonable, let me add that

**Review by ChristophWurst — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

File: apps/user_status/src/store/userStatus.js (hunk 1 above)
Comment (skjnldsv) [reply apps/user_status/src/store/userStatus.js:158]: @ChristophWurst you approved this event name?

Comment (ChristophWurst) [reply apps/user_status/src/store/userStatus.js:158]: what is the issue? the `.`?

**Pull request merged**